.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -42,9 +45,19 @@ jobs:
           submodules: true
       - name: Install Python requirements
         run: python3 -m pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
+      - name: Setting up Cache
+        run: |
+          mkdir -p /mnt/azureblob
+          python3 ./build_tools/ccache/edit_fuse_connection.py ${{ secrets.AZURE_CCACHE_CONTAINER_KEY }}
+          blobfuse2 mount --allow-other --config-file=./build_tools/ccache/fuse_connection2.yaml /mnt/azureblob/
+          export SCCACHE_DIR=/mnt/azureblob/ccache-container
+
       # TODO(#18238): add local or remote ccache
       - name: CMake - configure
         run: |
+          export SCCACHE_AZURE_CONNECTION_STRING=${{ secrets.SCCACHE_AZURE_CONNECTION_STRING  }}
+          export SCCACHE_AZURE_BLOB_CONTAINER="ccache-container"
```

Comment:
You can set `SCCACHE_AZURE_KEY_PREFIX` here to something matching the job name (e.g. `linux_x64_clang` here). I'll write up more thoughts about namespacing on the infra tracking issue.